src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/AddUsingsCodeActionResolver.cs

```diff
@@ -126,7 +126,9 @@ public override async Task<WorkspaceEdit> ResolveAsync(JObject data, Cancellatio
                     continue;
                 }
 
+#pragma warning disable CA1307 // Specify StringComparison
                 if (newUsingNamespace.CompareTo(usingDirectiveNamespace) < 0)
```

Comment:
Namespace comparisons should _not_ be "case-sensitive and culture-sensitive". This looks like a bug that should be fixed and not suppressed.
```suggestion
                if (string.CompareOrdinal(newUsingNamespace, usingDirectiveNamespace) < 0)
```